packages/inline-definition/src/InlineDefinition.story.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { css } from '@leafygreen-ui/emotion';
+import { H2, Body } from '@leafygreen-ui/typography';
+import InlineDefinition from '.';
+
+const shardDefinition =
+  'Sharding is a method for horizontally scaling across multiple replica sets by breaking up large datasets (e.g. partitioning) into smaller parts. Sharding is native to MongoDB.';
+
+const replicaSetDefintion =
+  'An Atlas replica set is a group of MongoDB servers with the same data to provide redundancy and high availability. ';
+
+storiesOf('InlineDefinition', module).add('Default', () => (
```

Comment:
Knobs for content and such would be nice here.

Reply:
Done!